roles/etcd/tasks/sync_etcd_master_certs.yml

```diff
@@ -13,6 +13,8 @@
     sync_file: "{{ item }}"
     sync_file_dir: "{{ etcd_cert_dir }}"
     sync_file_hosts: [ "{{ inventory_hostname }}" ]
+    sync_file_owner: kube
```

Comment:
is this correct? here is kube and in other files are vault ?

Reply:
sadly yes.. it's kube. I'm just keeping it here for consistency.